bcipy/tasks/rsvp/alert_tone_calibration.py

```diff
@@ -0,0 +1,56 @@
+"""Calibration Task that uses alert tones to help the user retain focus."""
+
+from bcipy.helpers.stimuli_generation import play_sound, soundfiles
+from bcipy.tasks.task import Task
+from bcipy.tasks.rsvp.calibration import RSVPCalibrationTask
+
+TASK_NAME = 'RSVP Alert Tone Calibration Task'
```

Comment:
I agree this should be a high static variable, but I think it should belong to the class and called via self.

Reply:
Sounds good, thanks.